Bio/SeqIO/FastaIO.py

```diff
@@ -158,6 +158,65 @@ def FastaTwoLineParser(source):
             handle.close()
 
 
+# used by fasta_ncbi_parser()
+# based on https://ncbi.github.io/cxx-toolkit/pages/ch_demo#ch_demo.id1_fetch.html_ref_fasta
+# those without a match in NCBI's or Uniprot's dbxref are commented out
+# NCBI's dbxref: https://www.ncbi.nlm.nih.gov/genbank/collab/db_xref/
+# Uniprot's dbxref: https://www.uniprot.org/docs/dbxref
+# RefSeq, Genbank & DDBJ not in NCBI's dbxref but in Uniprot's
+ncbi_identifiers = {
+    # "lcl": (('id',), "local"),
+    "bbs": (('id',), "GenInfo backbond seqid"),
```

Comment:
flake8 on TravisCI is failing,

```
$ flake8
./Bio/SeqIO/FastaIO.py:169:14: BLK100 Black would make changes.
./Bio/SeqIO/FastaIO.py:169:14: Q000 Remove bad quotes
...
```

The black style is double quotes rather than single quote characters for simple string literals. Try:

```
$ pip install black
$ black Bio/SeqIO/FastaIO.py
$ git commit -m "ran black" Bio/SeqIO/FastaIO.py
```

Reply:
thanks that worked